frontend/packages/dev-console/src/components/health-checks/ProbeForm.scss

```
@@ -0,0 +1,4 @@
+.odc-heath-check__probe-form {
```

Following BEM naming conventions a form is considered as block and not an element. http://getbem.com/naming/
```suggestion
.odc-heath-check-probe-form {
```